src/main/resources/eu_country_code.csv

```diff
@@ -0,0 +1,65 @@
+country code,description
```

Comment:
could be easier to read if we sort it either by country code or by formal name ?

Reply:
you right !